src/script/common/c_content.cpp

```diff
@@ -1315,12 +1315,12 @@ void pushnode(lua_State *L, const MapNode &n)
 }
 
 /******************************************************************************/
-void warn_if_field_exists(lua_State *L, int table,
-		const char *name, const std::string &message)
+void warn_if_field_exists(lua_State *L, int table, const char *fieldname,
+		const std::string &name, const std::string &message)
 {
-	lua_getfield(L, table, name);
+	lua_getfield(L, table, fieldname);
 	if (!lua_isnil(L, -1)) {
-		warningstream << "Field \"" << name << "\": "
+		warningstream << "Field \"" << fieldname << "\" on " << name << ": "
```

Comment:
even if not currently used I think this should handle the `name.empty()` case

Reply:
Handled in f7004fa75